packages/eslint-config/base.cjs

```diff
@@ -154,6 +146,7 @@ module.exports = {
       'error',
       { ignoreArrowShorthand: true },
     ],
+    '@typescript-eslint/no-import-type-side-effects': ['error'],
```

Comment:
Wanted to call this one out because it finally adds lint enforcement for:

```ts
import { type Thing } from './place'
// vs
import type { Thing } from './place'
```

Which could be really helpful for circular dependency issues, depending on how rollup and/or esbuild do or don't strip imports

Reply:
Yes! Nice, glad this is linted now